demos/embeddings/deep-graph-infomax-cora.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>I think we can just save the <code>gcn_acc</code> from the first section and not have to rerun with GCN? I.e. adjust the second "Extracting Embeddings and Logistic Regression" cell to end with:</p><p> </p><pre class="ql-syntax" spellcheck="false">gcn_acc = (y_pred == test_subjects).mean()

print(f"Test classification accuracy: {gcn_acc}")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>